docs/tutorials/multimodal/object_detection/detection_eval_yolov3_coco.md

```diff
@@ -0,0 +1,87 @@
+# AutoMM Detection - Evaluate Pretrained YOLOv3 on COCO Format Dataset
+:label:`sec_automm_detection_eval_yolov3_coco`
+
+In this section, our goal is to evaluate YOLOv3 model on COCO17 dataset in COCO format. We start with yolov3 because it's extremely fast and accurate, and is a good choice to deploy with strict time and computational restrictions.
+
+To start, import MultiModalPredictor:
+
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+
+We select the YOLOv3 with MobileNetV2 as backbone, 
```

Comment:
I think either in the very beginning or here, we should mention the number of parameters of the model and the speed of the model, so that users can know how fast it is. We should also provide such numbers for fasterrcnn and detr. Users can then choose which model to use based on these numbers. In terms of speed, we can benchmark the model on a p3.2x machine with 320x320 image resolution, then compute the latency (ms) and fps (frame per second).